formats/shared/definitions/multi_variant_pages.jsonnet

```diff
@@ -0,0 +1,28 @@
+{
+  transaction_variants: {
```

Comment:
Since this is only used in the `transaction` format, I wonder if it's better for the definition to live in the `transaction.jsonnet` file rather than its own file.

Reply:
I've moved the whole lot back into the `transaction` file since it's only used there.